docker-multinode/test/util.sh

```diff
@@ -64,7 +66,7 @@ start_tests() {
 
   # Compile minimal number of packages to run tests
   apt-get update
-  apt-get install -y make gcc
+  apt-get install -y build-essential
   cd ${K8S_LOCATION_PATH}
   go get -u github.com/jteeuwen/go-bindata/go-bindata
   make all WHAT=cmd/kubectl
```

Comment:
Please do `build/run.sh make WHAT="cmd/kubectl vendor/github.com/onsi/ginkgo/ginkgo test/e2e/e2e.test"`, so docker is used for compiling the binaries. That's the official way of building bins.

After that, go doesn't need to be downloaded